sdk/state/integration/helpers_test.go

```diff
@@ -27,35 +27,15 @@ func initAccounts(t *testing.T) (initiator Participant, responder Participant) {
 	t.Log("Initiator:", initiator.KP.Address())
 	t.Log("Initiator Escrow:", initiator.Escrow.Address())
 	{
-		err := retry(2, func() error { return fund(client, initiator.KP.FromAddress(), 10_000_0000000) })
+		err := retry(2, func() error { return createAccount(client, initiator.KP.FromAddress(), 10_000_0000000) })
```

Comment:
here we are creating the account and then next funding it

Reply:
this is also super helpful context! 💯